artisatomic/readqubdata.py

```diff
@@ -76,7 +100,14 @@ def read_adf04(filepath, flog):
         for _, row in qubupsilondf_alltemps.iterrows():
             lower = int(row["lower"])
             upper = int(row["upper"])
-            upsilon = float(row["upsT=5.01+03"].replace("-", "E-").replace("+", "E+"))
+            # W I and II rates are calculated at different temperatures
+            # Should be handled in a less approximate way in the future
+            if atomic_number == 74 and ion_stage == 1:
+                upsilon = float(row["upsT=5.80+03"].replace("-", "E-").replace("+", "E+"))
+            elif atomic_number == 74 and ion_stage == 2:
+                upsilon = float(row["upsT=4.00+03"].replace("-", "E-").replace("+", "E+"))
+            else:
+                upsilon = float(row["upsT=5.00+03"].replace("-", "E-").replace("+", "E+"))
```

Comment:
This changes the behaviour of the existing cobalt readin (upsT=5.01+03 changed to upsT=5.00+03). I didn't add a QUB cobalt test to GitHub, since the data is not public. I can try to make a cut-down version for testing to make sure that still works.

Reply:
I've added a QUB cobalt test now and updated your branch.